app/src/main/res/layout/activity_pager.xml

```diff
@@ -13,6 +13,7 @@
     <RelativeLayout
         android:layout_width="match_parent"
         android:layout_height="match_parent"
+        android:id="@+id/suneeti"
```

Comment:
suneeti ? Please user proper name.

Reply:
@mohitmanuja that was not added ! I pushed the code for the mentors to check as the Snackbar is still not visible. I will be pushing again with changes and correctness

Reply:
@mohitmanuja chck the issue ! I showed toast first then as Shubhankar said , I showed the snackbar

Reply:
@codesun Snackbar is visible on my device. Just hide the delete button when there is nothing to delete and use proper name for id.